client/types/disputes.d.ts

```diff
@@ -0,0 +1,51 @@
+// eslint-disable-next-line wpcalypso/import-docblock
+import { Order } from '@woocommerce/api';
+import { Charge } from './charges';
+
+interface Evidence {
+	[ key: string ]:
+		| string
+		| Record< string, boolean >
+		| Record< string, string >;
+	isUploading: Record< string, boolean >;
+	metadata: Record< string, string >;
+	uploadingErrors: Record< string, string >;
```

Comment:
@kalessil 
API response doesn't contain these 3 keys. 

Looking at the code it seems that we later add them and pass them along to other components with additional fields. But honestly, I'm lacking experience with codebase to figure out from where and why are they coming from.

Reply:
@dmvrtx I'm guessing these fields were taken from the same PR as others. Are we confident we actually need them? Unnecessary fields in definition could create a lot of confusion in the future.

Reply:
Let's drop them from this type. If a need arises later, there is always a way to create a derivative type with those fields.